api/openapi/v2/core-data.yaml

```diff
@@ -180,54 +194,65 @@ components:
       properties:
         config:
           description: "A string-ified representation of the service's configuration. For purposes of this specification, a string has been used since configuration structure differs from service to service."
```

Comment:
No longer `string-ified representation ` . How about `An object containing the service's configuration` ?

Reply:
The configuration content doesn't change for V2, but will change some each release, so having a link directly to the configuration section, which has a release in it, would require updating every release. Maybe just say something like `refer to Core Data's configuration documentation for more details at https://docs.edgexfoundry.org`???

Reply:
Requested changes made.